extensions/wikia/CommunityHeader/tests/WikiButtonsTest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+use \Wikia\CommunityHeader\Label;
+use \Wikia\CommunityHeader\WikiButton;
+use \CommunityHeader\WikiButtons;
+
+class WikiButtonsTest extends \WikiaBaseTest {
+	/**
+	 * $this->setUp() is called only after data providers are checked
+	 * We need the classes ready before that
+	 */
+	private function manualSetUp() {
+		global $IP, $wgAutoloadClasses;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$wgAutoloadClasses" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)